test/test.dart

```diff
@@ -399,9 +399,18 @@ String generateRandomRealmPath() {
 }
 
 final random = Random();
+String _getRandom(int len, String chars) => List.generate(len, (index) => chars[random.nextInt(chars.length)]).join();
```

Comment:
_getRandom as a method name that returns a `String` is not great imo. I suggest we revert this change

Reply:
I removed this method.